lms/djangoapps/instructor/views/api.py

```diff
@@ -1130,7 +1130,7 @@ def get_single_query(request, course_id, inclusion, query_type, state_type, stat
         }
         return JsonResponse(response_payload)
     else:
-       # 500 on all other unexpected status codes.
+        #500 on all other unexpected status codes.
```

Comment:
This properly moves the `#` over a single space, but it inadvertently removes the space after the `#`. Comments should begin with a single space character.